components/dynatrace_activegate/prod.tf

```diff
@@ -1,6 +1,7 @@
 module "dynatrace_activegate" {
   source         = "../../modules/dynatrace-activegate"
   instance_count = 3
+  network_zone   = "azure.cft"
```

Comment:
this probably won't apply on a running one, might need to scale it down to scale and scale it up, or click update to newest version in the portal

Reply:
will scale down then put it back up after build has run